src/EventStore.ClusterNode/ClusterNodeOptions.cs

```diff
@@ -353,6 +356,7 @@ public class ClusterNodeOptions : IOptions {
 
 			StatsPeriodSec = Opts.StatsPeriodDefault;
 
+			DeadMemberRemovalPeriodMin = Opts.DeadMemberRemovalTimeoutDefault;
```

Comment:
`Min` here could be confused for `Minimum` rather than `Minutes`.
Given that we don't have any other settings that use minutes, perhaps it would be better to use seconds here instead?
So `DeadMemberRemovalPeriodSec` instead?